plugin/import/import.view.js

```diff
@@ -0,0 +1,33 @@
+import {PluginView} from '../plugin.view';
+import {Command} from '@grid/core/command';
+import {upload} from '@grid/core/services/upload';
+import {readFile} from './read';
+
+export class ImportView extends PluginView {
+	constructor(model, context) {
+		super(...arguments);
+
+		const element = context.element;
+		const eventListener = context.eventListener;
```

Comment:
Kate can you put event call functionality to plugin itself? 
`.on(() =>$import.load()...`